src/bp/core/routers/auth.ts

```diff
@@ -35,10 +43,14 @@ export class AuthRouter implements CustomRouter {
   }
 
   getAuthConfig = async () => {
+    const config = await this.configProvider.getBotpressConfig()
+    const strategy = _.get(config, 'pro.auth.strategy', 'basic')
+    const authEndpoint = _.get(config, 'pro.auth.options.authEndpoint')
```

Comment:
What if pro is not configured because user is on community and doesn't care about pro ? is there a default value for this ?

Reply:
then strategy will be set to 'basic' and endpoint as undefined, which is the desired behaviour